assemble/src/main/scripts/create-jshell.sh

```diff
@@ -0,0 +1,74 @@
+#! /usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+function addAccumuloAPI() {
+  local srcDir="$1" api apiPath
+  # Validate each source directory before populating JShell-Init file
+  if [[ ! -d "$srcDir" ]]; then
+    echo "$srcDir is not a valid directory. Please make sure it exists."
+    exit 1
+  fi
+
+  # Extract API info from provided source directory
+  mapfile -t api < <(find "$srcDir" -type f -name '*.java' -print0 | xargs -0 -n1 dirname | sort -u)
+
+  # Load in API and format source directory into Java import statements
+  for apiPath in "${api[@]}"; do
+    echo "import ${apiPath##*/java/}.*" | tr / .
```

Comment:
Trailing semicolon isn't strictly necessary for JShell, but it's nice to have.

```suggestion
    echo "import ${apiPath##*/java/}.*;" | tr / .
```